Add an option to allow copying image indexes alone (#1511)
containers/skopeo: docs/skopeo-copy.1.md

Print usage statement

> **--multi-arch**

Reviewer: Also add the item to `completions/bash/skopeo`, please.

> If _source-image_ refers to an index of images for multiple architectures, this option controls what is copied. Default is system.

Reviewer: ```suggestion
Control what is copied if _source-image_ refers to a multi-architecture image. Default is system.
```

Options:
- system: Copy the image that matches the system architecture instead of the index
- all: Copy all of the images in the index and the index itself

> - index-only: Copy only the index itself

Reviewer: This should likewise talk about images instead of the details of the index/manifest list / … terminology (except for the index-only case).

> - index-only: Copy only the index itself

Reviewer: Add something like “… . Note that this usually fails unless the referenced per-architecture images are already present in the destination.”?
